src/EditorFeatures/CSharpTest/Classification/SemanticClassifierTests.cs

```diff
@@ -3917,11 +3918,8 @@ public async Task TestCreateWithBufferNotInWorkspace()
         var globalOptions = workspace.ExportProvider.GetExportedValue<IGlobalOptionService>();
 
         var provider = new SemanticClassificationViewTaggerProvider(
-            workspace.GetService<IThreadingContext>(),
-            workspace.GetService<ClassificationTypeMap>(),
-            globalOptions,
-            visibilityTracker: null,
-            listenerProvider);
+            workspace.GetService<TaggerHost>(),
```

Comment:
this pattern of passing these 5 items through became just passing along this new wrapper object (the 'TaggerHost').  Cleans up a lot of stuff.